StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Controllers/PayWithLinkViewController-WalletViewController.swift

```diff
@@ -35,14 +35,16 @@ extension PayWithLinkViewController {
             paymentPicker.supportedPaymentMethodTypes = viewModel.supportedPaymentMethodTypes
             paymentPicker.billingDetails = context.configuration.defaultBillingDetails
             paymentPicker.billingDetailsCollectionConfiguration = context.configuration.billingDetailsCollectionConfiguration
+            paymentPicker.linkAppearance = viewModel.linkAppearance
             return paymentPicker
         }()
 
         private lazy var mandateView = LinkMandateView(delegate: self)
 
         private lazy var confirmButton = ConfirmButton.makeLinkButton(
             callToAction: viewModel.confirmButtonCallToAction,
-            compact: viewModel.shouldUseCompactConfirmButton
+            compact: viewModel.shouldUseCompactConfirmButton,
+            linkAppearance: viewModel.linkAppearance
```

Comment:
Note: Other `ConfirmButton` configurations in this PR use `context.linkAppearance` directly here. I added this to the `viewModel` just to follow similar patterns here, but not opposed to replacing this line with:

```swift
linkAppearance: context.linkAppearance
```

if we’d like to minimize changes here.